runtime.txt

```diff
@@ -0,0 +1 @@
+python-3.6.1
```

Comment:
In what context is this file used?

Reply:
This was used for heroku cloud hosting. Removed in d534dbab7dc17bb83ec5b6937918c1963ca092c7